Makefile

```diff
@@ -20,11 +20,11 @@ test-docker:
 	@# run tests inside a docker container
 	docker run --rm -v $(PWD):/host -w /host --network="host" vapor/swift:5.2-bionic make test
 
-test-e2e: reset reconcile ingest
+test-e2e: db-reset reconcile ingest analyze
 	@# run import sequence test
 
 migrate:
-	$(VAPOR) run migrate
+	echo y | $(VAPOR) run migrate
```

Comment:
There may be a reason we wouldn't want to do this, but I can't think of one. Can you?